app/src/main/java/com/example/tmdb/screens/detail/DetailPage.kt

```diff
@@ -0,0 +1,112 @@
+package com.example.tmdb.screens.detail
+
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.example.tmdb.R
+
+
+class DetailPage{
+    @OptIn(ExperimentalFoundationApi::class)
+    @Composable
+fun DetailPage(visible : MutableState<Boolean>){
+        val reviews = remember{ mutableStateOf(true)}
+        val discussions = remember{ mutableStateOf(false)}
+    Scaffold(topBar = {
+        TopBar(visible = visible)
+    }
+
+    ){
+        LazyColumn(verticalArrangement = Arrangement.spacedBy(0.dp)) {
```

Comment:
Lazy components are used for large datasets, when you don't know how many items you'll need to display. This is not the case, implement this screens layout without this `LazyColumn`.